src/Diagnostics/FxCop/Core/Design/CA1052DiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.AnalyzerPowerPack.Utilities;
+
+namespace Microsoft.AnalyzerPowerPack.Design
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
```

Comment:
A doc comment here explaining the purpose of the rule and a remark about the merging of CA1052 and CA1053 would be good.